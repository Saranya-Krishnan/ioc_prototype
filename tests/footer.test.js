import React from 'react';
import Footer from '../views/src/components/footer';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router'

test('it displays the correct footer items', () =>{
    const clickTest = () => {
        return true;
    };
    const component = renderer.create(
        <MemoryRouter>
            <Footer clickFooterItem={'test'} Item={clickTest}></Footer>
        </MemoryRouter>
    );
    let c = component.toJSON();
    expect(c).toMatchSnapshot();
});