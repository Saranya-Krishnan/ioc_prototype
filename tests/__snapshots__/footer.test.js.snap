// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it displays the correct footer items 1`] = `
<div
  className="footer-wrapper"
>
  <div
    className="ui inverted vertical segment ioc-footer"
  >
    <div
      className="ui container center aligned"
    >
      <img
        alt={undefined}
        className="ui centered image"
        height={undefined}
        src={undefined}
        width={undefined}
      />
      <div
        className="ui small divided inverted link list"
        role="list"
      >
        <a
          href="/user"
          onClick={[Function]}
        >
          profile
        </a>
      </div>
    </div>
  </div>
</div>
`;
